roles/dds_cli/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: Install dds-cli
+  pip:
+    name: dds-cli
```

Comment:
I think it would be good to specify which version to install. On a related note: https://scilifelab.slack.com/archives/C0356S24TFA/p1645727630344029?thread_ts=1645716846.655839&cid=C0356S24TFA

Reply:
Done!

Reply:
Ah wait, Ina is releasing a new version soon.